RastenbergerAngularJS/src/main/webapp/foodoffers.js

```diff
@@ -0,0 +1,54 @@
+var app = angular.module("foodoffer-directives", []);
+
+app.directive("foodofferList", function () {
+    return {
+        restrict: "E",
+        templateUrl: "foodoffer-list.html",
+        controller: ["$http", "$log", function ($http, $log) {
+                $log.log("Getting all foodoffers from REST resource.");
+
+                var controller = this;
+                controller.foodoffers = [];
+
+                $http({
+                    method: "GET",
+                    url: "http://localhost:8080/RastenbergerRest/rs/foodoffers"
```

Comment:
Unfortunately the url is hard coded. I think you would need to adapt this, in order to run this successfully.